src/brs/http/GetState.java

```diff
@@ -53,7 +53,7 @@ JsonElement processRequest(HttpServletRequest req) {
     response.addProperty("lastBlock", blockchain.getLastBlock().getStringId());
     response.addProperty("cumulativeDifficulty", blockchain.getLastBlock().getCumulativeDifficulty().toString());
 
-    if (!"false".equalsIgnoreCase(req.getParameter("includeCounts"))) {
+    if ("true".equalsIgnoreCase(req.getParameter(INCLUDE_COUNTS_PARAMETER))) {
```

Comment:
This call makes the node to query for the balance of every account, now making this computation only if specifically asked.